### PR TITLE
Snack color

### PR DESCRIPTION
**Pull request by jslavitz**
Adds love (test) to #22489, first created by @noordawod.

### CODE REVIEW

File: packages/flutter/lib/src/material/snack_bar.dart [hunk 1]
```diff
@@ -81,12 +81,21 @@ class SnackBarAction extends StatefulWidget {
   /// The [label] and [onPressed] arguments must be non-null.
   const SnackBarAction({
     Key key,
+    this.labelColor,
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/material/snack_bar.dart]: I would call these textColor and disabledTextColor to match what we're passing them too

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/material/snack_bar.dart [hunk 2]
```diff
@@ -81,12 +81,21 @@ class SnackBarAction extends StatefulWidget {
...
     Key key,
+    this.textColor,
+    this.disabledTextColor,
     @required this.label,
     @required this.onPressed,
   }) : assert(label != null),
        assert(onPressed != null),
        super(key: key);
 
+  /// The button label color. If not provided, defaults to accentColor.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/snack_bar.dart]: `accentColor` should be hyperlinked (`[accentColor]`)

File: packages/flutter/lib/src/material/snack_bar.dart [hunk 3]
```diff
@@ -81,12 +81,21 @@ class SnackBarAction extends StatefulWidget {
...
+  final Color textColor;
+
+  /// The button disabled label color. This color is shown after the
+  /// snackBarAction is dismissed.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/snack_bar.dart]: similarly here

File: packages/flutter/test/material/snack_bar_test.dart [hunk 4]
```diff
@@ -296,6 +296,49 @@ void main() {
...
+    );
+
+    await tester.tap(find.text('X'));
+    await tester.pump(); // start animation
+    await tester.pump(const Duration(milliseconds: 750));
+
+    final Element actionTextBox = tester.element(find.text('ACTION'));
+    final Widget textWidget = actionTextBox.widget;
+    final DefaultTextStyle defaultTextStyle = DefaultTextStyle.of(actionTextBox);
+    if (textWidget is Text) {
```
Comment (Hixie) [inline packages/flutter/test/material/snack_bar_test.dart]: if it isn't text, should the test fail?

Comment (jslavitz) [reply packages/flutter/test/material/snack_bar_test.dart]: It will always be text.

File: packages/flutter/lib/src/material/snack_bar.dart (hunk 3 above)
Comment (jslavitz) [reply packages/flutter/lib/src/material/snack_bar.dart]: What is here? Don't think there's any named variable?

**Pull request merged**